config/config.php

```diff
@@ -3,13 +3,13 @@
 
 class dbconfig
 {
-    public static $emailBaseUrl = "http://smartkyc.ml";
+    public static $emailBaseUrl = "https://kyc-intel.herokuapp.com";
 
     /** @var string - Redis */
-    public $HOST     = 'localhost';
-    public $USER     = 'root';
-    public $PASSWORD = '';
-    public $DB_NAME  = 'smartkyc';
+    public $HOST     = 'ec2-52-20-36-181.compute-1.amazonaws.com';
+    public $USER     = 'h';
+    public $PASSWORD = 'pb4d210f5bc588b8177921b7c2467054a951758019920a0f595b5e6ce03b619cd';
```

Comment:
It's not best practice to push the server details to git. Please remove the credentials.